<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Maeva TECHER | publications</title>
  <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

  <link rel="shortcut icon" href="/assets/img/favicon.ico">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/publications/">
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        <strong>Maeva</strong> TECHER
    </span>
    

    <nav class="site-nav">
      <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

      <div class="trigger">
        <!-- Maeva do Science -->
        <a class="page-link" href="/">About</a>


        <!-- Pages -->
        <a class="page-link" href="/research/">Research</a>
        <a class="page-link" href="/publications/">Publications</a>
        <a class="page-link" href="/media/">Communication</a>
          
        <!-- Blog -->
        <!a class="page-link" href="/blog/">Blog<!/a>

        <!-- CV link -->
        <a class="page-link" href="/assets/pdf/CV_092020.pdf">CV</a>

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Scientific publications</h1>
    <h5 class="post-description">Publications by categories in reversed chronological order. Generated by jekyll-scholar.</h5>
  </header>

  <div class="img_row">
    <img class="col three left" src="/assets/img/banner_research.jpg" alt="" title="banner_research" />
</div>
<div class="col three caption">
</div>

  <article class="post-content publications clearfix">
    
<h3 class="year">Preprints</h3>
<ol class="bibliography">
<li>

<div id="toolkit">
  
    <span class="title">A toolkit for studying Varroa genomics and transcriptomics: Preservation, extraction, and sequencing library preparation</span>
    <span class="author">
      
          Hasegawa, N.,
          <em>Techer, M. A.</em>,
          and Mikheyev, A. S.

    </span>

    <span class="periodical">
    
      <em>biorxiv</em>
    
    
      Preprints
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.biorxiv.org/content/10.1101/2020.08.17.255083v1" target="_blank">HTML</a>]
  
  </span>



  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Background The honey bee parasite, Varroa destructor, is the leading cause of global honey bee population declines. In addition to being an obligate ectoparasitic mite, Varroa carries several viruses that infect honey bees and act as the proximal causes of colony collapses. Yet, until recently, the study of Varroa has been limited by the availability of genomic tools. Lab- and field-based methods exploiting such methods are still nascent. This study developed a set of methods for preserving Varroa DNA and RNA from the field to the lab and for processing them into sequencing libraries. We performed preservation testing experiments in which Varroa mites were immersed in TRIzol, RNAlater, and absolute ethanol, as well as preservation periods up to 21 days post-treatment to assess DNA and RNA integrity.</p>

    <p>Results For both DNA and RNA, mites preserved in TRIzol and RNAlater at room temperature degraded within 10 days post-treatment; mites preserved in ethanol at room temperature and 4°C remained intact through 21 days. Varroa mite DNA and RNA libraries were created and sequenced for ethanol preserved samples, 15 and 21 days post-treatment. All DNA sequences mapped to V. destructor at above 95% on average, while RNA sequences mapped to V. destructor, but also sometimes to high levels of the deformed wing virus, and other taxa.</p>

    <p>Conclusion Ethanolic preservation of field-collected mites is inexpensive, simple and allows them to be shipped and processed successfully in the lab for a wide variety of sequencing applications. It appears to preserve RNA from both Varroa and at least some of the viruses it vectors. </p>
  </span>
  
</div>
</li>

<li>

<div id="switch">
  
    <span class="title">The first steps toward a global pandemic: Reconstructing the demographic history of parasite host switches in its native range</span>
    <span class="author">
      
          <em>Techer, M. A.</em>,
          Roberts, J. M. K.,
          Cartwright, R. A.,
          and Mikheyev, A. S.

    </span>

    <span class="periodical">
    
      <em>biorxiv</em>
    
    
      Preprints
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.biorxiv.org/content/10.1101/2020.07.30.228320v1" target="_blank">HTML</a>]
  
  </span>



  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Host switching allows parasites to expand their niches. However, successful switching may require suites of adaptations and may decrease performance on the old host. As a result, reductions in gene flow accompany many host switches, driving speciation. Because host switches tend to be rapid, it is difficult to study them in real time and their demographic parameters remain poorly understood. As a result, fundamental factors that control subsequent parasite evolution, such as the size of the switching population or the extent of immigration from the original host, remain largely unknown. To shed light on the host switching process, we explored how host switches occur in independent host shifts by two ectoparasitic honey bee mites (Varroa destructor and V. jacobsoni). Both switched to the western honey bee (Apis mellifera) after it was brought into contact with their ancestral host (Apis cerana), ~70 and ~12 years ago, respectively. Varroa destructor subsequently caused worldwide collapses of honey bee populations. Using whole-genome sequencing on 44 mites collected in theirnative ranges from both the ancestral and novel hosts, we were able to reconstruct the known temporal dynamics of the switch. We further found that hundreds of haploid genomes were involved in the initial host switch, and, despite being greatly reduced, some gene flow remains between mites adapted to different hosts remains. Our findings suggest that while reproductive isolation may facilitate fixation of traits beneficial for exploitation of the new host, ongoing genetic exchange may allow genetic amelioration of inbreeding effects. </p>
  </span>
  
</div>
</li>
</ol>

<h3 class="year">2020</h3>
<ol class="bibliography">
<li>

<div id="varroa_review">
  
    <span class="title">Varroa destructor: A Complex Parasite, Crippling Honey Bees Worldwide</span>
    <span class="author">
      
        
          
            
              
                Traynor, K. S., 
                Mondet, F., 
                de Miranda, J. R., 
                <em>Techer, M.</em>,
                Kowallik, V., 
                Oddie, M. A. Y., 
                Chantawannakul, P., 
                & McAfee, A. 
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Trends in Parasitology</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.sciencedirect.com/science/article/pii/S147149222030101X" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>The parasitic mite, Varroa destructor, has shaken the beekeeping and pollination industries since its spread from its native host, the Asian honey bee (Apis cerana), to the naïve European honey bee (Apis mellifera) used commercially for pollination and honey production around the globe. Varroa is the greatest threat to honey bee health. Worrying observations include increasing acaricide resistance in the varroa population and sinking economic treatment thresholds, suggesting that the mites or their vectored viruses are becoming more virulent. Highly infested weak colonies facilitate mite dispersal and disease transmission to stronger and healthier colonies. Here, we review recent developments in the biology, pathology, and management of varroa, and integrate older knowledge that is less well known.</p>
  </span>
  
</div>
</li>

<li>


<div id="varroa_nigeria">
  
    <span class="title">A country-wide survey of Varroa destructor, an ectoparasitic mite of honey bees, in Nigeria: a preliminary report.</span>
    <span class="author">
      
        
          
            
              
                 Dukku, U. H., 
                 <em>Techer, M.</em>, 
                 & Vincent, S. N.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Journal of Apicultural Research,</em>
    
    
      2020
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.tandfonline.com/doi/abs/10.1080/00218839.2019.1664720" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Through a country-wide survey, the present study aimed at investigating the extent of the spread of Varroa destructor in Nigeria. To this end, a total of 122 Apis mellifera colonies from 13 localities were sampled in regions without previous assessment of the mite. Infestation and prevalence rates were estimated using the soap washing method. Varroa mites were found in 87% of the sampled colonies with a parasitic load ranging from 0.20 to 16.35 mites per 100 adult bees per colony. Given the presence of the mite with a very high prevalence in all the sampled localities and the fact that each of the states from which the mite is not yet reported is next to at least one occupied state, it is very likely that V. destructor spread throughout Nigeria. Future molecular analyses can evaluate whether V. destructor in Nigeria stems from a single source or additional introductions.</p>
  </span>
  
</div>
</li>
</ol>

<h3 class="year">2019</h3>
<ol class="bibliography">
<li>


<div id="varroa_genome">
  
    <span class="title">Divergent evolutionary trajectories following speciation in two ectoparasitic honey bee mites.</span>
    <span class="author">
      
        
          
            
              
                <em>Techer, M. A.</em>, 
                Rane, R. V., 
                Grau, M. L., 
                Roberts, J. M. K., 
                Sullivan, S. T., 
                Liachko, I., 
                Childers, A. K., 
                Evans, J. D., 
                & Mikheyev, A. S.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Communications Biology</em>
    
    
      2019
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://www.nature.com/articles/s42003-019-0606-0" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Multispecies host-parasite evolution is common, but how parasites evolve after speciating remains poorly understood. Shared evolutionary history and physiology may propel species along similar evolutionary trajectories whereas pursuing different strategies can reduce competition. We test these scenarios in the economically important association between honey bees and ectoparasitic mites by sequencing the genomes of the sister mite species Varroa destructor and Varroa jacobsoni. These genomes were closely related, with 99.7% sequence identity. Among the 9,628 orthologous genes, 4.8% showed signs of positive selection in at least one species. Divergent selective trajectories were discovered in conserved chemosensory gene families (IGR, SNMP), and Halloween genes (CYP) involved in moulting and reproduction. However, there was little overlap in these gene sets and associated GO terms, indicating different selective regimes operating on each of the parasites. Based on our findings, we suggest that species-specific strategies may be needed to combat evolving parasite communities.</p>
  </span>
  
</div>
</li>
</ol>


<h3 class="year">2018</h3>
<ol class="bibliography">
<li>



<div id="bee_genomics">
  
    <span class="title">Autosomal and Mitochondrial Adaptation Following Admixture: A Case Study on the Honeybees of Reunion Island.</span>
    <span class="author">
      
        
          
            
              
                 Wragg, D., 
                 <em>Techer, M. A.</em>, 
                 Canale-Tabet, K., 
                 Basso, B., 
                 Bidanel, J.-P., 
                 Labarthe, E., 
                 Bouchez, O., 
                 Le Conte, Y., 
                 Clémencet, J., 
                 Delatte, H., 
                 & Vignal, A. 
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Genome Biology and Evolution,</em>
    
    
      2018
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://academic.oup.com/gbe/article/10/1/220/4675325" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>The honeybee population of the tropical Reunion Island is a genetic admixture of the Apis mellifera unicolor subspecies, originally described in Madagascar, and of European subspecies, mainly A. m. carnica and A. m. ligustica, regularly imported to the island since the late 19th century. We took advantage of this population to study genetic admixing of the tropical-adapted indigenous and temperate-adapted European genetic backgrounds. Whole genome sequencing of 30 workers and 6 males from Reunion, compared with samples from Europe, Madagascar, Mauritius, Rodrigues, and the Seychelles, revealed the Reunion honeybee population to be composed on an average of 53.2 ± 5.9% A. m. unicolor nuclear genomic background, the rest being mainly composed of A. m. carnica and to a lesser extent A. m. ligustica. In striking contrast to this, only 1 out of the 36 honeybees from Reunion had a mitochondrial genome of European origin, suggesting selection has favored the A. m. unicolor mitotype, which is possibly better adapted to the island’s bioclimate. Local ancestry was determined along the chromosomes for all Reunion samples, and a test for preferential selection for the A. m. unicolor or European background revealed 15 regions significantly associated with the A. m. unicolor lineage and 9 regions with the European lineage. Our results provide insights into the long-term consequences of introducing exotic specimen on the nuclear and mitochondrial genomes of locally adapted populations.</p>
  </span>
  
</div>
</li>
</ol>

<h3 class="year">2017</h3>
<ol class="bibliography">
<li>


<div id="bee_archipelago">
  
    <span class="title">Genetic diversity and differentiation among insular honey bee populations in the southwest Indian Ocean likely reflect old geographical isolation and modern introductions.</span>
    <span class="author">
      
        
          
            
              
                 <em>Techer, M. A.</em>, 
                 Clémencet, J., 
                 Simiand, C., 
                 Turpin, P., 
                 Garnery, L., 
                 Reynaud, B., 
                 & Delatte, H.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>PloS One</em>
    
    
      2017
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0189234" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>With globalization the Western honey bee has become a nearly cosmopolitan species, but it was originally restricted to the Old World. This renowned model of biodiversity has diverged into five evolutionary lineages and several geographic “subspecies.” If Apis mellifera unicolor is indubitably an African subspecies endemic to Madagascar, its relationship with honey bees from three archipelagos in the southwest Indian Ocean (SWIO) hotspot of biodiversity is misunderstood. We compared recent mtDNA diversity data to an original characterization of the nuclear diversity from honey bees in the Mascarenes and Comoros archipelagos, using 14 microsatellites, but also additional mtDNA tRNALeu-cox2 analysis. Our sampling offers the most comprehensive dataset for the SWIO populations with a total of 3,270 colonies from 10 islands compared with 855 samples from Madagascar, 113 from Africa, and 138 from Europe. Comprehensive mitochondrial screening confirmed that honey bees from La Réunion, Mauritius, and Comoros archipelagos are mainly of African origin (88.1% out of 2,746 colonies) and that coexistence with European lineages occurs only in the Mascarenes. PCA, Bayesian, and genetic differentiation analysis showed that African colonies are not significantly distinct on each island, but have diversified among islands and archipelagos. FST levels progressively decreased in significance from European and African continental populations, to SWIO insular and continental populations, and finally among islands from the same archipelago. Among African populations, Madagascar shared a nuclear background with and was most closely related to SWIO island populations (except Rodrigues). Only Mauritius Island presented clear cytoplasmic disequilibrium and genetic structure characteristic of an admixed population undergoing hybridization, in this case, between A. m. unicolor and A. m. ligustica, A. m. carnica and A. m. mellifera-like individuals. Finally, global genetic clustering analysis helped to better depict the colonization and introduction pattern of honey bee populations in these archipelagos.</p>
  </span>
  
</div>
</li>

<li>


<div id="bee_mtDNA">
  
    <span class="title">Large-scale mitochondrial DNA analysis of native honey bee Apis mellifera populations reveals a new African subgroup private to the South West Indian Ocean islands.</span>
    <span class="author">
      
        
          
            
              
                 <em>Techer, M. A.</em>, 
                 Clémencet, J., 
                 Simiand, C., 
                 Preeaduth, S., 
                 Azali, H. A., 
                 Reynaud, B., 
                 & Hélène, D. 
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>BMC Genetics,</em>
    
    
      2017
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://bmcgenet.biomedcentral.com/articles/10.1186/s12863-017-0520-8" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Background
The South West Indian Ocean (SWIO) archipelagos and Madagascar constitute a hotspot of biodiversity with a high rate of endemism. In this area, the endemic subspecies A. m. unicolor has been described in Madagascar. It belongs to the African lineage, one of the four described evolutionary lineages in honey bees. Despite a long beekeeping tradition and several recorded European introductions, few studies have been carried out on the diversity and proportion of honey bee subspecies.

In order to identify and define which evolutionary lineages and potential sub-lineages are present in the SWIO, the COI-COII intergenic region and the ND2 gene of the mtDNA were sequenced in honey bee colonies from three archipelagos. An extensive sampling (n = 1184 colonies) was done in the Mascarene (La Réunion, Mauritius, Rodrigues), Seychelles (Mahé, Praslin, La Digue) and Comoros (Grande Comore, Mohéli, Anjouan, Mayotte) archipelagos. Islands genetic diversity was compared to newly sampled populations from Madagascar, continental African and European populations.

Results
African lineage haplotypes were found in all islands (except for Rodrigues). Madagascar, Comoros and Seychelles had 100% of A lineage, 95.5% in La Réunion and 56.1% in Mauritius. Among all African colonies detected in the SWIO, 98.1% (n = 633) of COI-COII haplotypes described the presence of the subspecies A. M. unicolor. Both genetic markers revealed i) a new private AI mitochondrial group shared by the SWIO archipelagos and Madagascar distant from continental populations; ii) the private African haplotypes for each island suggested diversity radiation in the archipelagos; iii) the detection of the Comoros archipelago as a possible contact area between insular and continental African populations.

The exotic European C and M lineages were only detected in the Mascarene archipelago, but striking differences of proportion were observed among islands. Merely 4.6% of European colonies were found in La Réunion whereas Mauritius cumulated 44%. Here, among the 84 observed COI-COII haplotypes, 50 were newly described including 13 which were private to the SWIO archipelagos and Madagascar. Similarly, 24 of the 34 found ND2 haplotypes were novel which included six haplotypes particular to the SWIO populations.

Conclusion
A new African subgroup was described in the SWIO region with mitochondrial genetic evidence that A. m. unicolor is the indigenous subspecies of the archipelagos surrounding Madagascar.</p>
  </span>
  
</div>
</li>
</ol>

<h3 class="year">2016</h3>
<ol class="bibliography">
<li>


<div id="bee_seychelles">
  
    <span class="title">Genetic diversity of the honeybee ( Apis mellifera L.) populations in the Seychelles archipelago.</span>
    <span class="author">
      
        
          
            
              
                 <em>Techer, M. A.</em>, 
                 Clémencet, J., 
                 Simiand, C., 
                 Portlouis, G., 
                 Reynaud, B., 
                 & Delatte, H. 
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Insect Conservation and Diversity,</em>
    
    
      2016
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/icad.12138" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>In the South‐West Indian Ocean, the honeybee Apis mellifera is found on several islands including the Seychelles archipelago. This archipelago is located 1120 km North of Madagascar, where the endemic African subspecies A. m. unicolor occurs. The genetic diversity of the honeybee populations in the Seychelles islands has never been studied, yet this species interacts with highly endemic and indigenous flora.
A total of 186 honeybee colonies from the three main islands: Mahé, Praslin and La Digue were collected. In addition, 107 individuals from Madagascar (A. m. unicolor) and 49 from Italy (A. m. ligustica) were analysed as reference populations. The maternal lineages were assessed using PCR‐RFLP (n = 342) and sequencing (n = 121) of the mtDNA COI–COII intergenic region. Intra‐Seychelles nuclear genetic diversity and structure were analysed using 15 microsatellites while comparison with reference populations was done using 14 loci.
All Seychellian colonies had mtDNA sequences characteristic of the African evolutionary lineage. Two sub‐lineages were detected: AI sub‐lineage (A1) was dominant (96.7%) on all islands and mostly represented by the subspecies A. m. unicolor, while Z sub‐lineage was observed in six colonies from two islands. No mtDNA characteristic of imported European lineages was detected.
Nuclear genetic diversity was high and structured, suggesting restricted gene flow between islands of the archipelago. High nuclear similarities were found among the Seychellian and A. m. unicolor populations, yet significant genetic differentiation was observed. The A. m. ligustica reference population was highly differentiated from the Seychellian honeybee populations.</p>
  </span>
  
</div>
</li>
</ol>

<h3 class="year">2015</h3>
<ol class="bibliography"><li>


<div id="bee_rodrigues">
  
    <span class="title">Genetic characterization of the honeybee (Apis mellifera) population of Rodrigues Island, based on microsatellite and mitochondrial DNA.</span>
    <span class="author">
      
        
          
            
              
                 <em>Techer, M. A.</em>, 
                 Clémencet, J., 
                 Turpin, P., 
                 Volbert, N., 
                 Reynaud, B., 
                 & Delatte, H. 
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Apidologie,</em>
    
    
      2015
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s13592-014-0335-9" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Apis mellifera is present in Rodrigues, an island in the South-West Indian Ocean. The history of the established honeybee population is poorly known, and its biodiversity has never been studied. In this study, maternal origins of A. mellifera in Rodrigues have been assessed with the DraI test and sequencing of the mitochondrial COI-COII region. Nuclear genetic diversity was investigated with 18 microsatellite markers. A total of 524 colonies were sampled from 16 beekeepers in 20 sites. The Rodrigues population was characterized by the absence of the African lineage and presence of three European haplotypes from the C lineage. C1 haplotype was the most frequent (81.3 %), followed by C2 (18.0 %) (characteristics of A. m. ligustica and carnica, respectively) and a new haplotype, C1-Rod (0.7 %). No genetic structure was detected, and genetic diversity was comparable to continental populations. According to approximate Bayesian computation (ABC) analyses, bottleneck scenarios are most likely to have occurred.</p>
  </span>
  
</div>
</li>
<li>


<div id="bee_madagascar">
  
    <span class="title">Genetic diversity of the endemic honeybee: Apis mellifera unicolor (Hymenoptera: Apidae) in Madagascar.</span>
    <span class="author">
      
        
          
            
              
                 Rasolofoarivao, H., 
                 Clémencet, J., 
                 <em>Techer, M. A.</em>, 
                 Ravaomanarivo, L. H. R., 
                 Reynaud, B., 
                 & Delatte, H.
              
            
          
        
      
    </span>

    <span class="periodical">
    
      <em>Apidologie</em>
    
    
      2015
    
    </span>
  

  <span class="links">
  
    [<a class="abstract">Abs</a>]
  
  
  
    [<a href="https://link.springer.com/article/10.1007/s13592-015-0362-1" target="_blank">HTML</a>]
  
  
  
  
  
  </span>

  <!-- Hidden abstract block -->
  
  <span class="abstract hidden">
    <p>Apis mellifera unicolor is a tropical honeybee endemic of Madagascar. Comprehensive knowledge about its mitochondrial and nuclear genetic diversity and structuration was our main purpose. Samples of worker bees were collected from 867 colonies in 76 sites in Madagascar and 1 reference population in South Africa. PCR-restriction fragment length polymorphism (RFLP) and sequencing were used to reveal variability in the COI–COII mtDNA region. Seventeen microsatellite loci were used for studying the nuclear diversity. Three PCR-RFLP profiles were observed, among which 99.4 % belonged to A1 haplotype, 0.2 % to a new A haplotype, and 0.4 % to A4 haplotype. In microsatellite analysis, moderate genetic diversity values were found for Madagascar, together with low mean number of alleles ranging from 2.47 to 3.88 compared to South Africa. Bayesian clustering assignment methods and principal component analysis (PCA) separated populations into two genetic clusters which matched with geographic areas. Several hypotheses are discussed regarding to the low genetic diversity of A. m. unicolor in its native range.</p>
  </span>
  
</div>
</li>
</ol>


  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper">
    &copy; Copyright 2020 Walter Dempsey.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
  </div>

</footer>


    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


<!-- Load KaTeX -->
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js"></script>
<script src="/assets/js/katex.js"></script>




<!-- Include custom icon fonts -->
<link rel="stylesheet" href="/assets/css/fontawesome-all.min.css">
<link rel="stylesheet" href="/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXXX', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
